Problem2-MinHeap.py

```diff
@@ -0,0 +1,43 @@
+# A Min-Heap is a complete binary tree in which the value in each internal node is less than or equal to the values in the children of that node.
+# Mapping the elements of a heap into an array is trivial: if a node is stored a index k, then its left child is stored at index 2k + 1 and its right child at index 2k + 2.
+
+# Time complexity: insert(): O(log(n)) ; getMin(): O(1) ; extractMin(): O(log(n))  
+# Space complexity: O(n) ; storing all elements
+
+from heapq import heappush, heappop, heapify
```

Comment:
The goal of a design question is to be implemented from scratch without using libraries, please rework this problem without heapq library.

Reply:
Okay, I will update this, thanks for letting me know!